### PR TITLE
Schema dev

### PR DESCRIPTION
**Pull request by duckyou**
```bash
----------------------------------------------- benchmark: 1 tests -----------------------------------------------
Name (time in ms)          Min       Max      Mean  StdDev    Median     IQR  Outliers     OPS  Rounds  Iterations
------------------------------------------------------------------------------------------------------------------
test_old_schema       179.1040  182.2439  181.0547  1.1816  181.4068  1.2067       2;0  5.5232       5           1
test_new_schema       14.9814   15.5025   15.1083   0.1533  15.0423   0.1440       2;2  66.1890      19          1
------------------------------------------------------------------------------------------------------------------
```

### CODE REVIEW

**Comment by lk-geimfari** [comment]
@duckyou Check Travis report, please.

**Comment by lk-geimfari** [comment]
@duckyou You did so great job! This implementation works much faster (extremely) than previous.

```
cProfile.run('schema.create(iterations=10000)')
```

An old implementation:
```
         7972958 function calls in 27.306 seconds

   Ordered by: standard name

   ncalls  tottime  percall  cumtime  percall filename:lineno(function)
    10000    0.238    0.000   27.280    0.003 .test.py:9(<lambda>)
        1    0.007    0.007   27.306   27.306 <string>:1(<module>)
    10000    0.020    0.000    0.122    0.000 _strptime.py:29(_getlang)
    10000    0.273    0.000    0.511    0.000 _strptime.py:302(_strptime)
    # ...
    # ...
```

New implementation:
```
         2246634 function calls (2246507 primitive calls) in 2.446 seconds

   Ordered by: standard name

   ncalls  tottime  percall  cumtime  percall filename:lineno(function)
    10000    0.127    0.000    2.430    0.000 .test.py:9(<lambda>)
        1    0.000    0.000    0.000    0.000 <frozen importlib._bootstrap>:119(release)
        1    0.000    0.000    0.000    0.000 <frozen importlib._bootstrap>:159(__init__)
        # ...
        # ...
```

That's really good!

File: mimesis/schema.py [hunk 1]
```diff
@@ -63,24 +64,37 @@ def __call__(self, name: Optional[str] = None,
...
-                if name in dir(provider):
-                    method = getattr(provider, name)
-                    result = method(**kwargs)
-                    if key and callable(key):
-                        return key(result)
-                    return result
-        else:
-            raise UnsupportedField(name if not providers
-                                   else '{}.{}'.format(providers[0], name))
+        def tail_parser(tails, obj):
```
Comment (lk-geimfari) [inline mimesis/schema.py]: Maybe:
```
def tail_parser(tails: str, obj: Any) -> Any:
    # ...
```

Comment (duckyou) [reply mimesis/schema.py]: @lk-geimfari Right, forgot about it. Thanks! 👍 
Notify me, plz, when you do full review 😃

Comment (lk-geimfari) [reply mimesis/schema.py]: @duckyou That's all. As I can judge everything is excellent. Just annotate this function and let's merge it!

**Pull request merged**